posts/Davide/2017-10-16-hibernate-ogm-5-2-Beta1-released.adoc

```diff
@@ -0,0 +1,78 @@
+= "Hibernate OGM 5.2 Beta1 is out"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Another month, another release: http://hibernate.org/ogm/[Hibernate OGM] 5.2 Beta1 is ready!
+
+The main goal for this release is to support Infinispan 9.1 and allow the creations of caches
+when they are missing.
+
+If you need to upgrade from a version before 5.2, you can find help on the
+https://developer.jboss.org/wiki/HibernateOGMMigrationNotes[migration notes].
+
+All the changes are described in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31400[release notes].
+
+== Infinispan remote automatic cache creation
+
+By default, Hibernate OGM expects the caches you defined via the mapping and the configuration
+to already exit on the Infinispan server. At start up, an exception is thrown if one of the caches
+is not found.
+ 
+With this version is now possible to have Hibernate OGM create the caches on startup if they are
```

Comment:
Maybe replace "With this version is" with "With this version, it is"?

Reply:
Between the two I prefer the original phrasing: subj ("with this version") + verb
It seems simpler.